monitor/validators.js

```diff
@@ -65,76 +65,99 @@ async function main(bridgeMode) {
   const foreignBridgeValidators = new web3Foreign.eth.Contract(BRIDGE_VALIDATORS_ABI, foreignValidatorsAddress)
 
   logger.debug('calling foreignBridgeValidators getValidatorList()')
-  const foreignValidators = await getValidatorList(foreignValidatorsAddress, web3Foreign.eth, {
+  const foreignValidators = (await getValidatorList(foreignValidatorsAddress, web3Foreign.eth, {
     from: MONITOR_FOREIGN_START_BLOCK,
     to: foreignBlockNumber,
     logger
-  })
+  })).map(web3Foreign.utils.toChecksumAddress)
```

Comment:
While testing, I found that on some networks the addresses are returned in upper or lower case, making the comparison of the validator addresses to incorrectly detect that validators were not on the other network. To fix this, the addresses are now converted to checksum addresses

Reply:
well done